kuma/wiki/models.py

```diff
@@ -811,6 +809,9 @@ def revert(self, revision, user, comment=None):
         revision.created = datetime.now()
         revision.creator = user
         revision.save()
+        if revision.document.original == self:
```

Comment:
This is the same comparison as exists in the code now. Can we preserve it to minimize changes since this will already be a pretty low-level refactor? I can add a `TODO` line above to change though.